corehq/tabs/tabclasses.py

```diff
@@ -1625,6 +1626,10 @@ def sidebar_items(self):
 
         items.append((_('Manage Enterprise'), enterprise_views))
 
+        if BillingAccount.sms_billable_report_visible(self.domain):
+            items.extend(EnterpriseReportDispatcher.navigation_sections(
+                request=self._request, domain=self.domain))
```

Comment:
I don't think this is necessarily a blocking comment, but it seems strange that the visibility of the sms billable report determines the visibility of _all_ reports that use the `EnterpriseReportDispatcher`. I understand that this is only the sms billable report right now, but if someone adds another enterprise specific report in the future, this will likely need to change. Does that sound right?

Reply:
Yep, since sms billable report was only report, I hid it entirely (it actually hides the whole tab group "Manage Billing Details"), so if we need to add more reports to show in that group, we'll have to implement a different method,  maybe something like making "visibility" an attribute of the reports themselves so they aren't dispatched to the page.